config/rbac/role_binding.yaml

```diff
@@ -10,3 +10,17 @@ subjects:
 - kind: ServiceAccount
   name: controller-manager
   namespace: system
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: manager-rolebinding-cluster-admin
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
```

Comment:
@zqq454224016 our arcadia do not require `cluster-admin` . Please minimize this to the least permission

Reply:
manager-role does not have sufficient permissions

Reply:
then you should add the missing permissions to manager-role.